src/gpodder/util.py

```diff
@@ -1234,7 +1234,7 @@ def url_add_authentication(url, username, password):
     return urllib.parse.urlunsplit(url_parts)
 
 
-def urlopen(url, headers=None, data=None, timeout=None, **kwargs):
+def urlopen(url, headers=None, data=None, timeout=None, method="GET", **kwargs):
```

Comment:
Needed to make a POST request, so I added this functionality to urlopen. Should be non-breaking, as the parameter is added